package.json

```diff
@@ -19,14 +19,13 @@
     "test-backend": "env-cmd -f tests/.test-env jest --coverage --maxWorkers=4",
     "test-backend:watch": "env-cmd -f tests/.test-env jest --watch",
     "test:frontend": "npm --prefix frontend test",
-    "build": "npm run build-backend && npm run build-frontend",
-    "build-backend": "tsc -p tsconfig.build.json",
-    "build-frontend": "webpack --config webpack.prod.js",
-    "build-frontend-dev": "webpack --config webpack.dev.js",
-    "build-frontend-dev:watch": "webpack --config webpack.dev.js --watch",
+    "build": "npm run build:backend && npm run build:frontend",
+    "build:frontend": "npm run --prefix frontend build",
+    "build:backend": "tsc -p tsconfig.build.json",
     "start": "node -r dotenv/config dist/backend/src/app/server.js",
     "dev": "docker-compose up --build",
```

Comment:
I think it might not be very intuitive DX if `npm run dev` doesn't start up everything we need. is there a way to arrange the scripts to make that possible? i.e. to make this something like `docker-compose up --build & npm run --prefix frontend start`?

Reply:
Done in dcd3aab